sat/recipe.py

```diff
@@ -63,13 +63,13 @@ def __init__(self, version, vcs_repo, vcs_branch):
         """Create a new HPCSoftwareRecipe object.
 
         Args:
-            version (str): the recipe version string
+            version (packaging.version.Version): the recipe version string
```

Comment:
it looks like it should still be a `str` as an arg, but gets set to a `packaging.version.Version` in the attribute.

Reply:
Ah, I misread `Args` as `Attributes`. I'll change that back.